apache/yunikorn-k8shim
# [YUNIKORN-1828] unify plugin.yaml with other deployment example files

### What is this PR for?
changed items:

- add some description: `deployments/scheduler/README.md`
- `deployments/scheduler/plugin.yaml` remove RBAC, health check, load balancing
- remove `deployments/scheduler/scheduler-load.yaml`
- The load balance part is independent: `deployments/scheduler/service.yaml`
- `deployments/scheduler/yunikorn-configs.yaml` changed to follow [default-configmap](https://yunikorn.apache.org/docs/user_guide/service_config#default-configmap)
- `deployments/scheduler/yunikorn-rbac.yaml` namespace change to default to facilitate testing

### What type of PR is it?

### Todos

### What is the Jira issue?
https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-1828

### How should this be tested?
1 Install kind
2 sudo kubectl create -f yunikorn-rbac.yaml
3 sudo kubectl create configmap yunikorn-configs --from-file=yunikorn-configs.yaml
4 sudo kubectl apply -f plugin.yaml
5 check yunikorn web

### Screenshots (if appropriate)

### Questions:
